src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2Request.java

```diff
@@ -187,6 +197,11 @@ private void errorResult(Throwable e, HTTP2SampleResult res) {
     protected void sample(URL url, String method, HTTP2Connection http2Connection,
                           HTTP2SampleResult sampleResult) {
 
+        String urlStr = url.toString();
+        if (log.isDebugEnabled()) {

```

Comment:
Maybe we can change this log declaration to something like `Logger log = LoggerFactory.getLogger(HTTP2Request .class);` and then this validation won't be needed.